provider/git-retriever/src/main/java/org/arquillian/algeron/pact/provider/loader/git/ContractsGitLoader.java

```diff
@@ -58,6 +58,7 @@ public String getName() {
     @Override
     public void configure(Map<String, Object> configuration) {
         this.contractsGit = new ExternallyConfiguredContractsGit(configuration);
+        this.gitOperations = new GitOperations();
```

Comment:
How about putting this in the default constructor? I think that it has more sense so the initialization always occurs.